utils/blog/api-helpers.ts

```diff
@@ -0,0 +1,23 @@
+export function getStrapiURL(path = '') {
+	return `${process.env.NEXT_PUBLIC_STRAPI_API_URL || 'http://localhost:1337'}${path}`;
```

Comment:
This `localhost` url is for testing env right? we should remove it in prod

Reply:
done